src/main/scala/Day09.scala

```diff
@@ -0,0 +1,70 @@
+import scala.io.*
+object Day09 extends App:
+
+  val day: String = this.getClass.getName.drop(3).init
+  val start1: Long = System.currentTimeMillis
+
+  val pattern = "(\\d+) players; last marble is worth (\\d+) points".r
```

Comment:
nice - regex works indeed in pattern matching - personally - prefer string matching with explicit casting as regex are not easily readable by a lot of programmers - your code is being read an order of magnitude more times by others than by you - independent of that.. me like that you know how to match on regex-es